resolwe_bio/descriptors/unified-descriptor.yml

```diff
@@ -60,80 +60,80 @@
     - name: non_clinical_information
       label: Non-clinical information
       group:
-        - name: cell_line__cell_line_name
+        - name: cell_line_cell_line_name
```

Comment:
Drop the group name here, so just `cell_line_name` is retained. Do the same in the related field.